gradlew

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
+
+#
+# Copyright 2015 the original author or authors.
```

Comment:
Who is the original author of this script? Even if it's Apache-licensed, the original copyright has to be included